apps/chat/src/store/models/models.epics.ts

```diff
@@ -375,13 +375,16 @@ const updateRecentModelsEpic: AppEpic = (action$, state$) =>
     ignoreElements(),
   );
 
-const getModelsSuccessEpic: AppEpic = (action$) =>
+const getModelsSuccessEpic: AppEpic = (action$, state$) =>
   action$.pipe(
     filter(ModelsActions.getModelsSuccess.match),
     switchMap(({ payload }) => {
-      const defaultModelId = payload.models.find(
-        (model) => model.isDefault,
-      )?.id;
+      const overlayDefaultModelId =
+        SettingsSelectors.selectOverlayDefaultModelId(state$.value);
+
+      const defaultModelId = overlayDefaultModelId
+        ? undefined
```

Comment:
why do you return `undefined` here?

Reply:
We need it to skip to skip assignment of default model and array.find() run.